src/update_engine/postinstall_runner_action.cc

```diff
@@ -55,7 +73,7 @@ void PostinstallRunnerAction::PerformAction() {
   // Runs the postinstall script asynchronously to free up the main loop while
   // it's running.
   vector<string> command;
-  command.push_back(temp_rootfs_dir_ + kPostinstallScript);
+  command.push_back((identical_partitions ? "/usr" : temp_rootfs_dir_) + kPostinstallScript);
```

Comment:
I don't think this is an issue but I feel like pointing out that CompletePostInstall() will still try to umount `temp_rootfs_dir`.

Reply:
Yes, the destructor runs but has no error reporting, but good to state that this syscall will be done and fail